pkgs/development/python-modules/pydantic-settings/default.nix

```diff
@@ -14,7 +14,7 @@
 let
   self = buildPythonPackage rec {
     pname = "pydantic-settings";
-    version = "2.5.2";
+    version = "2.7.1";
```

Comment:
Is this absolutely necessary? Because that must go to staging and probably causes regressions

Reply:
Yes, MCP needed this version to build unfortunately.

Reply:
Try `pythonRelaxDeps = [ "pydantic-settings" ]` instead.
Upstream also relaxed the requirement https://github.com/modelcontextprotocol/python-sdk/commit/0d48a4494fc60d350ed1b10a88edc8b06f716d65

Reply:
Nice, it works !